src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -127,34 +155,55 @@ const PathwaysList: FC<PathwaysListProps> = ({ evaluatedPathways, callback, serv
 const PathwaysListElement: FC<PathwaysListElementProps> = ({
   evaluatedPathway,
   criteria,
-  callback
+  callback,
+  selected
 }) => {
   const classes = useStyles();
   const pathway = evaluatedPathway.pathway;
   const pathwayCtx = usePathwayContext();
   const [isVisible, setIsVisible] = useState<boolean>(false);
+  const { patientRecords, setPatientRecords } = usePatientRecords();
+  const patient = usePatient().patient as Patient;
+  const client = useFHIRClient();
 
   const chevron: IconProp = isVisible ? faChevronUp : faChevronDown;
 
   function toggleVisible(): void {
     setIsVisible(!isVisible);
   }
 
+  const pathwayElementClass = selected
```

Comment:
I'm also a little confused why we've split up our styles this way.

Reply:
I think `pathway-element` and `title` were styled in the component for theming so I added `selected-pathway-element` and `selected-title` to change the properties to stay consistent. Do you think I should move the newly added ones into the css module?

Reply:
Yeah I think it's confusing to split up the styling like this. Theming is done from the `styles/theme.tsx` file. Do we do this in other files? Thoughts, @radamson or @dehall?

Reply:
I see what's going on after staring at it for a bit.  It would be nice to consolidate the styles here, but I don't think we need to do it for every file in this PR.

**Old way of doing styles:**
Most of our components are styled using separate SASS style sheets that are imported.

```typescript
import styles from './PathwaysList.module.scss'

...

<div className = {styles.pathwaysStyle} />
```

**New way of doing styles:**
The new way of doing styles involves using JSS in the same component file, instead of SASS.

In https://github.com/mcode/pathways/pull/26 we started using material-ui which offers some nice utilities to help with styles and theming (which is really just a thin layer over JSS under the hood).

The nice thing here is it makes it really easy to apply the app theme through `makeStyles`.

Also looking at https://github.com/mcode/pathways/pull/26/files I think you're the one who originally added it this way (i.e. using both `classes` and `styles`) 🙂 .

Reply:
@noranda sounds good!  I created PATHWAYS-237 to move all the styles over and put in the description that all the JSS/makeStyles should be done in separate files from the component.  Good catch on this; it's an important task.